core/src/main/scala/org/genivi/sota/core/PackagesResource.scala

```diff
@@ -120,7 +123,13 @@ class PackagesResource(resolver: ExternalResolverClient, db : Database,
         pkg <- db.run(Packages.create(newPkg))
       } yield StatusCodes.NoContent
 
-      resultF.pipeToBus(messageBusPublisher)(_ => PackageCreated(ns, pid, description, vendor, signature))
+      val event = PackageCreated(ns, pid, description, vendor, signature)
+
+      resultF.flatMap { _ =>
+        AutoInstall.packageCreated(updateService, event)
+      }
```

Comment:
This future is ignored, ie, we never install any callbacks or combine it with other future, so the result will be ignore, if it fails we'll never know.